client/my-sites/customer-home/components/home-content/index.jsx

```diff
@@ -154,7 +156,11 @@ const HomeContent = ( {
 		);
 	}
 
-	if ( layout?.view_name === 'VIEW_FOCUSED_LAUNCHPAD' && ! focusedLaunchpadDismissed ) {
+	if (
+		layout?.view_name === 'VIEW_FOCUSED_LAUNCHPAD' &&
+		! focusedLaunchpadDismissed &&
+		! isMvpOnboarding
```

Comment:
I think adding client-side logic for showing/hiding the focused launchpad is a bad idea. @paulopmt1 previously did a refactor where we moved all the logic to the backend because we were introducing too many inconsistencies.

The logic on the backend for whether to show the focused launchpad or not lives here.
https://github.a8c.com/Automattic/wpcom/blob/d015ec61c62de4c800aeac8469e5b8fb27065401/wp-content/lib/home/views.php#L943

You can request a user's experiment assignment in PHP like this: PCYsg-Frn-p2

We should want to use the `get_user_assignment()` function instead, since we don't want to assign users to a cohort just because they load My Home. We should just check if the user happens to have been assigned to a cohort.

Reply:
That makes sense, and thanks for the details, Phill!

Here's a PR with the suggested implementation: 180929-ghe-Automattic/wpcom

Please let me know if this is what you had in mind. 🙏